stable/stash/values.yaml

```diff
@@ -47,3 +49,6 @@ apiserver:
   enableAdmissionWebhook: false
   # CA certificate used by main Kubernetes api server
   ca:
+
+# Send usage events to Google Analytics
+enableAnalytics: true
```

Comment:
I'm not sure how people will react to Chart defaults being "share my usage pattern". I'd say, default to false and let the user decide. At the very least, make a note about it in the README.

Reply:
We want this to be default true. If users don't want that they can set false.

Reply:
I agree with @timstoop. This should be disabled by default.